pages/en/android-environment/deploying-apps.md

```diff
@@ -91,16 +95,47 @@ adb install [path to your APK]
 
 If you can't use the method described above and need to push your app from another device, you can connect the device to ADB using [USB](#connect-to-adb-over-usb) or a [network address](#connect-to-adb-over-a-network).
 
-To push your APK from another device into the Chromebook, you must start your Chrome OS in [developer mode](https://chromium.googlesource.com/chromiumos/docs/+/master/developer_mode.md) so that you can configure the Chromebook and push apps from the host machine. Follow this steps to get into [developer mode](/{{locale.code}}/productivity/experimental-features#developer-mode).
+### Connect to ADB over a network {: #adb-ip}
+
+1. Make sure you [enabled ADB debugging.](#enable-adb-debug)
+
+Get the IP address of your Chromebook:
+
+1.  Click the clock in the bottom-right area of the screen.
+1.  Click the gear icon.
+1.  Click the network type you are connected to (Wi-Fi or Mobile data) then the name of the network.
+1.  Take note of the IP Address.
+
+Connect to your Chromebook:
+
+1.  Return to your development machine and use ADB to connect to your Chromebook using its IP address:
+
+    ```bash
+    adb connect <ip_address>:22
+    ```
+
+1.  On your Chromebook, click Allow when prompted whether you want to allow the debugger. Your ADB session is established.
+
+#### Troubleshooting ADB debugging over a network
```

Comment:
We may also want to mention that some office networks prohibit these kinds of connections (like the Google office does). In which case, the developer will need to either set up their own router or phone hotpost to connect both their development machine and chromebook to. It may help to include steps to diagnose that this is indeed the issue, but I honestly am not sure how one would determine this.

Reply:
Understandable if we don't want to tackle this advice for this PR, but we should create a bug instead then.